docs/app/components/theme-selector/theme-selector.component.less

```diff
@@ -0,0 +1,17 @@
+@import "../../assets/styles/variables.less";
+
+.theme-selector-button {
+    height: 50px;
+    font-size: 0.9375rem;
+    border: none;
+    padding: 15px;
+    background-color: transparent;
+
+    ux-icon[name="chevron-down"] {
```

Comment:
Does it need to be this specific? If we were to to replace this with a different icon we would need to update this too. Would just `ux-icon { ... }` be ok instead here?